docs/user_guide/datasets/acs_public_coverage.rst

```diff
@@ -0,0 +1,281 @@
+         4. Never served in the military
+         
+   *  - ANC
+      - Ancestry recode:
+         1. Single
+         2. Multiple
+         3. Unclassified
+         4. Not reported
+         8. Suppressed for data year 2018 for select PUMAs
+
+   *  - NATIVITY
+      - Nativity:
+         1. Native
+         2. Foreign born
+         
+   *  - DEAR
+      - Hearing difficulty:
+         1. Yes
+         2. No
+         
+   *  - DEYE
+      - Vision difficulty:
+         1. Yes
+         2. No
+         
+   *  - DREM
+      - Cognitive difficulty:
+         1. Yes
+         2. No
+         
+   *  - PINCP
+      - Total annual income per person as an integer between -19997 and 4209995
+        US dollars. Loss of $19998 or more is coded as -19998. Income of
+        $4209995 or more is coded as 4209995.
+
+   *  - ESR
+      - Employment status recode:
+         1. Civilian employed, at work
+         2. Civilian employed, with a job but not at work
+         3. Unemployed
+         4. Armed forces, at work
+         5. Armed forces, with a job but not at work
+         6. Not in labor force
+         
+   *  - ST
+      - State code: Please see data dictionary at `ACS PUMS documentation
+         <https://www.census.gov/programs-surveys/acs/microdata/documentation.2018.html>`_
+         for the full list of state codes.
+
+   *  - FER
+      - Gave birth to child within the past 12 months:
+         1. Yes
+         2. No
+
+   *  - RAC1P
+      - Race code
+         1. White alone
+         2. Black or African American alone
+         3. American Indian alone
+         4. Alaska Native alone
+         5. American Indian and Alaska native tribes specified; or American
+            Indian or Alaska Native, not specified and no other races
+         6. Asian alone
+         7. Native Hawaiian and Other Pacific Islander alone
+         8. Some Other Race alone
+         9. Two or More races
+
+
+The target label is given by PUBCOV, which can be used for a binary
+classification task.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 7 30
+   :stub-columns: 1
+
+   *  - Column name
+      - Description
+
+   *  - PUBCOV
+      - Public health coverage, with PUBCOV == 1 if the individual has public
+        health coverage, else 0
+
+
+.. _acs_public_coverage_discussion:
+
+Discussion
+^^^^^^^^^^
+
+As Ding et al. [1]_ notes, algorithmic (un)fairness must be evaluated in
+context of the intervention, and the effects of different interventions vary
+widely by state. Similarly, datasets should be carefully designed for the
+specific prediction task in mind.
+
+Therefore, to investigate how viable ACSPublicCoverage is as a fairness
+benchmark dataset, we must first ask what real-world tasks might require a
+prediction of whether an individual is covered by public health insurance.
+Usually, the entity that needs this information (e.g. the government checking
+for eligibilty in a social program) will simply ask the individual directly.
```

Comment:
I would expect that eligibility is based on a predetermined policy, does the government need to ask individuals directly or will they just determine eligibility based on provided information?

Reply:
do we have to clarify?